stream/BidirectionalStream.ts

```diff
@@ -62,24 +63,31 @@ export class BidirectionalStream {
         const responseQueue = this._responseCollector.queue(requestId);
         if (batch) this._dispatcher.dispatch(request);
         else this._dispatcher.dispatchNow(request);
-        return (await responseQueue.take() as Transaction.Res);
+        const value = await responseQueue.take() as Transaction.Res;
+        this._responseCollector.remove(requestId);
+        return value;
```

Comment:
remove a single rpc request immediately when it returns

Reply:
It's OK to remove the response collector even if `responseQueue.take` throws, right? I mean, the request will be dead in a ditch by that point. If so, I think this code would be more elegant:
```suggestion
        try {
            return await responseQueue.take() as Transaction.Res;
        } finally {    
            this._responseCollector.remove(requestId);
        }
```